apps/commudle-admin/src/app/feature-modules/hackathon-control-panel/components/hackathon-control-panel-basic-form/hackathon-control-panel-basic-form.component.ts

```diff
@@ -4,8 +4,10 @@ import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { ActivatedRoute, Router } from '@angular/router';
 import { HackathonService } from 'apps/commudle-admin/src/app/services/hackathon.service';
 import { EHackathonLocationType, EParticipateTypes, IHackathon } from 'apps/shared-models/hackathon.model';
-import { Subscription } from 'rxjs';
 import { faArrowRight, faFileImage, faLink } from '@fortawesome/free-solid-svg-icons';
+import { ICommunity } from 'apps/shared-models/community.model';
+import { ICommunityGroup } from 'apps/shared-models/community-group.model';
```

Comment:
let's use @commmudle/shared-models for ICommunity and SeoService from @commudle/shared-service